### PR TITLE
Remove RspecResources references - batch 1

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**
#5644 follow up

Batch replace for
```
[Rr]ule = DiagnosticDescriptorBuilder.GetDescriptor(DiagnosticId, MessageFormat, RspecStrings.ResourceManager
[Rr]ule = DescriptorFactory.Create(DiagnosticId, MessageFormat
```

### CODE REVIEW

File: analyzers/src/SonarAnalyzer.VisualBasic/Rules/VariableUnused.cs [hunk 1]
```diff
@@ -32,7 +32,7 @@ namespace SonarAnalyzer.Rules.VisualBasic
     [DiagnosticAnalyzer(LanguageNames.VisualBasic)]
     public sealed class VariableUnused : VariableUnusedBase
     {
-        private static readonly DiagnosticDescriptor rule = DiagnosticDescriptorBuilder.GetDescriptor(DiagnosticId, MessageFormat, RspecStrings.ResourceManager);
+        private static readonly DiagnosticDescriptor rule = DescriptorFactory.Create(DiagnosticId, MessageFormat);
```
Comment (Corniel) [inline analyzers/src/SonarAnalyzer.VisualBasic/Rules/VariableUnused.cs:35]: You keep the lowecase variations lowercase?

**Review by Corniel — APPROVED** [review]

File: analyzers/src/SonarAnalyzer.VisualBasic/Rules/VariableUnused.cs (hunk 1 above)
Comment (pavel-mikula-sonarsource) [reply analyzers/src/SonarAnalyzer.VisualBasic/Rules/VariableUnused.cs:35]: Long-term: No, we'll fix them.

For this PR (and the batch 2): I do not fix anything to keep the PR as small as possible. Also as regular as possible with no sideeffects.

Comment (Corniel) [reply analyzers/src/SonarAnalyzer.VisualBasic/Rules/VariableUnused.cs:35]: Logical choice. Just checking.

**Review by costin-zaharia-sonarsource — APPROVED** [review]
LGTM!

**Pull request merged**